From: Andreas Beckmann <anbe@debian.org>
Date: Mon, 23 Mar 2015 11:54:35 +0100
Subject: Use hardening CPPFLAGS

---
 GNUmakefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/GNUmakefile b/GNUmakefile
index 7bf7dca..dfe1554 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -41,7 +41,7 @@ endif
 CC?=		gcc
 CFLAGS?=	-O2 -pipe
 CFLAGS+=	-g -Wall
-CPPFLAGS=	-DREGEX
+CPPFLAGS+=	-DREGEX
 CPPFLAGS+=	-D_GNU_SOURCE
 CPPFLAGS+=	$(BSD_CPPFLAGS) -D__dead=__dead2
 LIBS=		$(CURSES_LIBS) $(BSD_LIBS)
